vfront_verilog_parser/src/preproc.rs

```diff
@@ -145,13 +145,25 @@ impl<'sm, 'a> Preproc<'sm, 'a> {
         self.ifdef_chunk_level = entry.1;
     }
 
+    /// Emits a diagnostic for nested block comments.
+    fn check_block_comment(&mut self, Token { src, ..}: Token<'sm>) {
```

Comment:
destructuring within argument list of a proper function (as opposed to a closure), while valid, feels highly unusual (and causes the argument to be nameless in the generated documentation) — just write `token.src` instead